WordPress/src/test/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModelTest.kt

```diff
@@ -170,13 +185,30 @@ class SiteCreationDomainsViewModelTest {
      * Verifies the UI state for after the user enters a non-empty query which results in multiple domain suggestions.
      */
     @Test
-    fun verifyNonEmptyUpdateQueryUiStateAfterResponseWithMultipleResults() = testWithSuccessResponse {
-        viewModel.start(null, SEGMENT_ID)
-        viewModel.updateQuery(MULTI_RESULT_DOMAIN_FETCH_QUERY.first)
-        val captor = ArgumentCaptor.forClass(DomainsUiState::class.java)
-        verify(uiStateObserver, times(3)).onChanged(captor.capture())
-        verifyVisibleItemsContentUiState(captor.thirdValue, showClearButton = true)
-    }
+    fun verifyNonEmptyUpdateQueryUiStateAfterResponseWithMultipleResults() =
+            testWithSuccessResponse {
+                viewModel.start(null, SEGMENT_ID)
+                viewModel.updateQuery(MULTI_RESULT_DOMAIN_FETCH_QUERY.first)
+                val captor = ArgumentCaptor.forClass(DomainsUiState::class.java)
+                verify(uiStateObserver, times(3)).onChanged(captor.capture())
+                verifyVisibleItemsContentUiState(captor.thirdValue, showClearButton = true)
+            }
+
+    /**
+     * Verifies the UI state for after the user enters a non-empty query which results in multiple domain suggestions.
+     */
+    @Test
+    fun verifyDomainUnavailableUiStateAfterResponseWithMultipleResults() =
```

Comment:
We might want to consider testing only that "domain unavailable" ui state is present, so the test has only a single reason to fail. Wdyt?

Reply:
Yes, that makes alot of sense. It makes it easier to follow. Resolved here in d01fc2c